src/mavm.rs

```diff
@@ -37,7 +40,7 @@ impl Label {
 
     pub fn avm_hash(&self) -> Value {
         match self {
-            Label::Func(sid) => Value::avm_hash2(
+            Label::Func(sid) | Label::Closure(sid) => Value::avm_hash2(
```

Comment:
Should the hash of a closure include the captured values?  Or maybe include the pre-built call frame (which includes the captured values)?

(And if we change this, do we want to change how functions are hashed, so that a function is equivalent to a closure with no captures?)

Reply:
So this is something that's a bit interesting. As it currently stands, func & closure label hashes are not unique. In fact, the vast majority of funcs share hashes with each other. This is because we are currently assigning them based on the local `StringId`of the func. After `xlating` they are differentiated, but this scheme is dangerous and hacky.

In the next PR #577, I make func & closure label mavm hashes based on the rust-hash of their file & names, ensuring they are unique across all stages of compilation and between versions of ArbOS. I think we should adopt the new scheme, accepting this one as a temporarily equivalent to the first that also happens to include closure labels.